x/autopilot/keeper/airdrop.go

```diff
@@ -24,15 +28,33 @@ func (k Keeper) TryUpdateAirdropClaim(
 		return errors.New("packet forwarding param is not active")
 	}
 
+	// verify packet originated on a registered host zone
+	if packet.GetDestPort() != transfertypes.PortID {
```

Comment:
I could go either way. Does feel a bit safer to pull it upstream, but the grabbing of the host zone does feel a bit more specific to each autopilot route.

That said, I think I still probably lean towards pulling it up and just passing the host zone as a param. Wdyt?

Are just to clarify, are you suggesting moving both the port and channel check or just the port?

Reply:
I'm in favor of moving this directly above the `routingInfo` switch in `OnRecvPacket`, just so future routes will have to explicitly decide "I don't want to restrict this to host zones", instead of that being the default option

Reply:
Sorry - I'm only suggesting we move the port check up!

Reply:
we could also just scratch the transfer port check altogether. It shouldn't even be possible to route to this callback if there was a non-transfer channel, and I don't think it gives us any additional security properties

Reply:
chatted offline, leaving it in the current place for now considering autopilot has the potential to be used for non-transfer packets